process_monitor_unix.py

```diff
@@ -89,8 +89,12 @@ def get_exit_status(self):
         return self.exit_status
 
     def stop_target(self):
-        os.kill(self.pid, signal.SIGKILL)
-        self.alive = False
+        try:
+            os.kill(self.pid, signal.SIGKILL)
+        except OSError:
+            return False
```

Comment:
Fantastic! This method didn't formerly return anything, so what do you think of dropping the `return False`? The return value is nowhere used in boofuzz; are you using it?